### PR TITLE
[ACA-1230] Manage versions from right panel tab.

### PR DESCRIPTION
**Pull request by suzanadirla**

### CODE REVIEW

File: src/app/ui/overrides/_adf-info-drawer.scss [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
\ No newline at end of file
+}
+
+.adf-manage-versions-empty,
+.adf-manage-versions-no-permission {
+    margin: 24px;
+    color: rgba(0, 0, 0, 0.54);
+    text-align: justify;
+
+    & &-icon {
```
Comment (pionnegru) [inline src/app/ui/overrides/_adf-info-drawer.scss]: double `& &`

File: src/app/ui/overrides/_adf-info-drawer.scss [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+        display: block;
```
Comment (pionnegru) [inline src/app/ui/overrides/_adf-info-drawer.scss]: why do we need to specify `display` property?

File: src/app/ui/overrides/_adf-info-drawer.scss [hunk 3]
```diff
@@ -1,3 +1,16 @@
...
+        font-size: 48px;
+        margin: 0 auto 32px auto;
```
Comment (pionnegru) [inline src/app/ui/overrides/_adf-info-drawer.scss]: I believe for mat-icon size, these value should be the same
```scss
&-icon {
     width: $icon-size;
     height: $icon-size;
     font-size: $icon-size;
}
```

File: src/app/ui/overrides/_adf-info-drawer.scss (hunk 1 above)
Comment (suzanadirla) [reply src/app/ui/overrides/_adf-info-drawer.scss]: It looks strange, I know, but that was intentional. For more clarity, this is now changed :)

**Pull request merged**